README.md

```
@@ -1,259 +1,193 @@
-- [사용 기술 및 환경](#-사용-기술-및-환경)
-- [프로젝트 소개](#-프로젝트-소개)
-- [아키텍쳐 구성도](#-아키텍쳐-구성도)
-- [ERD](#-ERD)
-- [기술적 고민 V1](#-기술적-고민-v1)
-- [개선사항 V2](#-개선-사항-v2)
-
-<br/>
-
-## 🧑‍💻 인원 소개
+## 인원 소개
 |                                                                조영무                                                                |                                        정석현                                        |                                                                                                              
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)